index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile First Project</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="logo">My Logo</a>
+      <nav class="hamburger">
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+      </nav>
+    </header>
+
+    <section class="main-container">
+      <div class="title">
+        <h1>This is Mert Glad to see you here!</h1>
+      </div>
+      <dev class="intro">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+        <p class="connect"><a href="#">LET'S CONNECT</a></p>
+
+        <div class="social-media">
+          <ul class="social-media-icons">
+            <li>
+              <a href="https://twitter.com/aranelluinwe">
+                <img class="icons" src="Images/twitter.png" alt="twitter logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/mert-kantaroglu/">
+                <img
+                  class="icons"
+                  src="Images/Linkedin.png"
+                  alt="linkedin logo"
+              /></a>
```

Comment:
- [Optional] Dear @mertkantaroglu, One major thing to be implemented here (Which is only optional because it has not been mentioned to you before) is the font type for your design. As you can see down your font is not the same as the design provided. Please Update this style accordingly. This will give the recruiter confidence you can work with UX/UI designers.
- [ ] Dear @mertkantaroglu, Good job making the most of the requested changes. although as the previous code reviewer mentioned please update the margins (top, bottom, left, right) defined in the design. you can use `margin: (value)px (value)px (value)px (value)px`
- [ ] One more thing is, even though you made a change to your `let's connect` text it still does not look closely the same as the design provided. please update your style accordingly. 

### Figma design 
![Capture2](https://user-images.githubusercontent.com/69077061/205226915-e7147366-c089-4e32-8f36-974194729e11.PNG)


### Your design 
![image](https://user-images.githubusercontent.com/69077061/205226574-4ad06786-38cf-4db0-a882-b9270a72d9f6.png)